src/sidebar/RoutingResults.tsx

```diff
@@ -59,6 +62,22 @@ function RoutingResult({ path, isSelected }: { path: Path; isSelected: boolean }
     )
 }
 
+// Used in RoutingResults.tsx and NavBar.ts for development purposes
+export function isFakeRequested() {
+    var query = window.location.search.substring(1);
+    console.log(query)
+    var vars = query.split("&");
+    console.log(vars)
+    for (var i=0;i<vars.length;i++) {
+        console.log(vars[i]) 
+        if(vars[i].includes("fake")) {
+            console.log("found fake")
+            return true;
+        }
+    }
+    return(false);
```

Comment:
We do not need to manually parse the url. We can use a URL object instead.

Furthermore it is important to remove all unnecessary logging from production code.